src/test/resources/test_data/policy_conditions/policyV2_1.json

```diff
@@ -949,7 +949,7 @@
                       "type": "text",
                       "label": "Enter the other frequency",
                       "name": "alternativeReportingFrequency",
-                      "case": "capitalise",
+                      "case": null,
```

Comment:
we decided to not fix this and keep the existing behaviour, as it didn't make sense to title case user provided durations